gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -66,106 +73,185 @@ def __init__(self, seed=None, config_file='config_files/config_example.json', co
         if not self.config['pickup_put_interaction']:
             self.action_names = tuple([action_name for action_name in self.action_names if 'Pickup'
                                        not in action_name and 'Put' not in action_name])
+        if self.config.get('put_interaction'):
```

Comment:
Can we simplify all these gets in one single for loop? 
`self.action_names = list(ALL_POSSIBLE_ACTIONS.copy())
for action_name in self.action_names:
    if action_name.startswith('Look') and not self.config.get('lookupdown_actions, True) or
    if action_name.startswith('Move') and not self.config.get('moveupdown_actions, True):
        self.action_names.remove(acton_name)
self.action_names = tuple(self.action_names)